models/credmark/algorithms/value_at_risk/var.py

```diff
@@ -1,15 +1,63 @@
 from credmark.cmf.model import Model
 from credmark.cmf.model.errors import ModelRunError
 
+from credmark.cmf.types import PriceList, Price
+
+
 from models.credmark.algorithms.value_at_risk.dto import (
     VaRHistoricalInput,
+    PortfolioVaRInput,
 )
 
 from models.credmark.algorithms.value_at_risk.risk_method import calc_var
 
 import numpy as np
 
 
+@Model.describe(slug='finance.var-portfolio-historical',
+                version='1.1',
+                display_name='Value at Risk - for a portfolio',
+                description='Calculate VaR based on input portfolio',
+                input=PortfolioVaRInput,
+                output=dict)
+class VaRPortfolio(Model):
+    def run(self, input: PortfolioVaRInput) -> dict:
+        portfolio = input.portfolio
+
+        pls = []
+        pl_assets = set()
+        for pos in portfolio:
```

Comment:
can we expand variables? I know it's longer, but it makes it easier to understand

Reply:
so `for position in portfolio`

Reply:
ok. renamed.